proselint/checks/glaad/terms.py

```diff
@@ -10,21 +10,31 @@
 categories: writing
 ---
 
-This check looks for improper terms related to LGTQ issues. The
-New York Times and Associated Press have also adopted this style guide.
+This check looks for possibly offensive terms related to LGBTQ issues and
+makes more acceptable recommendations. TheNew York Times and
+Associated Press have also adopted this style guide.
 
 """
 from proselint.tools import memoize, preferred_forms_check
 
 
 @memoize
 def check(text):
-	"""Suggest preferred forms given the reference document."""
-	err = "glaad.misc"
-	msg = "Possibly offensive term. Use '{}' instead of '{}'."
-	
-	glaad_terms = [
-	    [],
-	]
-	
-	return preferred_forms_check(text, glaad_terms, err, msg, ignore_case=False)
\ No newline at end of file
+    """Suggest preferred forms given the reference document."""
+    err = "glaad.terms"
+    msg = "Possibly offensive term. Consider using '{}' instead of '{}'."
+
+    list = [
+        ["gay man",            ["homosexual man"]],
+        ["gay men",            ["homosexual men"]],
+        ["lesbian",            ["homosexual woman"]],
+        ["lesbians",            ["homosexual women"]],
+        ["gay people",            ["homosexual people"]],
+        ["gay couple",            ["homosexual couple"]],
+        ["sexual orientation",           ["sexual preference"]],
+        ["gay lives",            ["homosexual lifestyle", "gay lifestyle"]],
+        ["out",            ["admitted homosexual", "avowed homosexual"]],
```

Comment:
This replacement has the same issue:

"The chairman of my Official Board was an admitted homosexual."

GLAAD recommends "openly gay", which I think would be better here. (This would also require changing the matched text from "admitted homosexual" to "an admitted homosexual".)